Arm/Memory/Separate.lean

```diff
@@ -328,6 +328,12 @@ theorem BitVec.not_le_eq_lt {a b : BitVec w₁} : (¬ (a ≤ b)) ↔ b < a := by
   rw [BitVec.le_def, BitVec.lt_def]
   omega
 
+theorem mem_separate'_comm (h : mem_separate' a an b bn) :
+    mem_separate' b bn a an := by
+  have := h.omega_def
+  apply mem_separate'.of_omega
+  omega
+
 /-#
 This is a theorem we ought to prove, which establishes the equivalence
```

Comment:
Just got reminded of this TODO theorem: `mem_separate_of_mem_separate'` -- let's add an issue to keep track of this.

Reply:
done: https://github.com/leanprover/LNSym/issues/137